tests/integration/kubernetes/k8s-policy-job.bats

```diff
@@ -9,10 +9,6 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
```

Comment:
If these tests start passing, we should re-enable this other one too: https://github.com/kata-containers/kata-containers/commit/a8ccd9a2ac7b91841adb5e5440314194755f32f0

Reply:
Just reverted that commit. I'm pretty confident at this point, all previous runs (listed in description) have been green.